utils/vaults.json

```diff
@@ -1029,10 +1029,10 @@
     "WANT_ADDR": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
     "WANT_SYMBOL": "WETH",
     "COINGECKO_SYMBOL": "weth",
-    "VAULT_STATUS": "active",
+    "VAULT_STATUS": "withdaw",
```

Comment:
The PR makes it looks like I'm changing Fresh Beets but this is actually changing "Enter the reactor," the tokemak eth strat